components/DocSearch.tsx

```diff
@@ -70,7 +92,9 @@ const DocSearch: FunctionComponent = () => {
       <Icon name="search" color="inherit" />
       <input
         id="algolia-avo-search"
-        placeholder="Search the documentation"
+        placeholder={
+          isLoading ? 'Loading search...' : 'Search the documentation'
```

Comment:
The text replacement when loading the page is slightly jarring. Could we avoid that somehow?

Reply:
Hmm can you elaborate?

Reply:
Ahh spotted! yes we can, one moment

Reply:
Fixed in 37834ec (or the jarring moved to only when the script tag is slow)

Reply:
Nice!